rainforest/tests.go

```diff
@@ -525,3 +587,27 @@ func (c *Client) UpdateTest(test *RFTest) error {
 	}
 	return nil
 }
+
+// UpdateWisp updates existing test's wisp and title on RF, requires WispJson struct to be prepared to upload using helpers
+func (c *Client) UpdateWisp(wisp *WispJson) error {
+	if wisp.TestID == 0 {
+		return errors.New("Couldn't update the test TestID not specified in RFTest")
+	}
```

Comment:
Backend does blow up, but I would prefer to give a better error message than `Can't find test with id 0`.

My reasoning for putting it here was to preemt the backend error and translate to a more informative message, but I agree it would be better placed elsewhere. I'll move it.

Reply:
Actually this was copy-pasta from from the `UpdateTest` function and I assume the intention of the original author was to provide a better error message. I'll leave this for the cleanup PR as well.